.github/ISSUE_TEMPLATE/FEATURE_REQUEST.yml

```diff
@@ -0,0 +1,52 @@
+name: Feature Request
+description: A suggestion for a new feature
+labels: [enhancement]
+body:
+- type: markdown
+  attributes:
+    value: "## Thank you for contributing to Swift-DocC!"
+- type: markdown
+  attributes:
+    value: |
+      Before you submit your issue, please replace each textarea
+      below with information about your proposed feature.
+- type: markdown
+  attributes:
+    value: "## Feature Request"
```

Comment:
I think this "Feature Request" heading is a bit redundant with the title of the template- what do you think?
<img width="775" alt="Screen Shot 2022-07-01 at 10 16 28 AM" src="https://user-images.githubusercontent.com/6750147/176940839-fde7c533-0148-4068-b809-d41e48fb35f6.png">
```suggestion
```

Reply:
+1